sphinx_ncs_theme/static/js/ncs.js

```diff
@@ -76,30 +76,6 @@ function NCS () {
     }
   };
 
-  /*
-   * Updates the dropbox of nRF Connect SDK versions displayed below the
-   * current development version, and links to the same page currently being
-   * browsed in those earlier releases.
-   */
-  state.updateVersionDropDown = function() {
-    const ncs = window.NCS;
-    const versions = ncs.versions.VERSIONS;
-    const by_version = ncs.versions.COMPONENTS_BY_VERSION[ncs.current_version] ||
-                       ncs.versions.COMPONENTS_BY_VERSION["latest"];
-
-    // Update dropdown text
-    const ncs_version = by_version["ncs"];
-    $("#ncsversion").text(`v${ncs_version}`);
-
-    // Update dropdown content
-    $.each(versions, function(_, v) {
-      if (v !== ncs.current_version) {
-        let link = `<a class="dropdown-item" href=${ncs.url_root + v}/${ncs.current_page}>${v}</a>`;
-        $("div.dropdown-menu").append(link);
-      }
-    });
```

Comment:
Are you sure this is dead code? I think this is what is populating the dropdown. The html classes `dropdown-menu` and `ncsversion` can still be found in the live documentation.

Reply:
right, I deleted too much!